Client/Client.Qt/Style/StyleBasic.cpp

```diff
@@ -93,19 +100,23 @@ namespace
 int  px0  = 0;
 int  px2  = 2;
 int  px3  = 3;
+int  px4  = 4;
 int  px5  = 5;
 int  px8  = 8;
 int  px10 = 10;
+int  px12 = 12;
 int  px14 = 14;
 int  px15 = 15;
 void initPxValues(int scale)
 {
     px0  = ConvertScale(0, scale);
     px2  = ConvertScale(2, scale);
     px3  = ConvertScale(3, scale);
+    px4  = ConvertScale(4, scale);
     px5  = ConvertScale(5, scale);
     px8  = ConvertScale(8, scale);
     px10 = ConvertScale(10, scale);
+    px12 = ConvertScale(12, scale);
     px14 = ConvertScale(14, scale);
     px15 = ConvertScale(15, scale);
```

Comment:
maybe, using some dynamic structure would be better then declaration of so many variables?

Reply:
No.
Only possible container for this is map, that may require from us to add additional check on valid data